components/modal/index.zh-CN.md

```diff
@@ -71,7 +71,7 @@ title: Modal
 | icon | 自定义图标（3.12.0 新增） | string\|ReactNode | `<Icon type="question-circle">` | 3.12.0 |
 | iconType | 图标类型（3.12.0 后废弃，请使用 `icon`） | string | `question-circle` |  |
 | mask | 是否展示遮罩 | Boolean | true | 3.13.0 |
-| maskClosable | 点击蒙层是否允许关闭 | Boolean | `false` |  |
+| maskClosable | 点击蒙层是否允许关闭 | Boolean | `true` |  |
```

Comment:
- `maskClosable` is `true` indeed by default for `Modal.method()`.

Reply:
I think that it can be close in mask when `maskClosable` is `true`.

So it's `true` in `Modal Component` case, it's `false` in `Modal.method()`

Reply:
All right, this issue makes no sense now, because it has been corrected.

![image](https://user-images.githubusercontent.com/35673748/64617432-0aaaa980-d411-11e9-8a8e-699a4e5bd84a.png)